packages/application-generic/src/decorators/user-session.decorator.ts

```diff
@@ -0,0 +1,38 @@
+import { createParamDecorator, UnauthorizedException } from '@nestjs/common';
+import * as jwt from 'jsonwebtoken';
+
+// eslint-disable-next-line @typescript-eslint/naming-convention
+export const UserSession = createParamDecorator((data, ctx) => {
+  let req;
+  if (ctx.getType() === 'graphql') {
```

Comment:
It is tech debt but I wonder why we keep having this.

Reply:
Good question